learn/cookbooks/running_production.mdx

```diff
@@ -197,132 +191,40 @@ server {
-
-#### 5.2.1. Option A: Certbot
-
-Using certbot in your Linux server is very straightforward. This tool will generate a free SSL/TLS certificate for your domain name, and automatically handle its installation on your server. The certbot documentation contains detailed instructions for many operating systems and servers, but we will follow the instructions for [Certbot on Debian with Nginx](https://certbot.eff.org/instructions?ws=nginx&os=debianbuster).
-
-First of all, install the packages on your system:
+First, install the required packages on your system:
 
 ```bash
 sudo apt-get install certbot python-certbot-nginx -y
 ```
 
-Let's run the Certbot script to be guided through the installation process:
+Next, run certbot:
 
 ```bash
 certbot --nginx
 ```
 
-Enter your email address, agree to the Terms and Conditions, and input your domain name. You will then be prompted with these options:
-
-```bash
-Please choose whether or not to redirect HTTP traffic to HTTPS, removing HTTP access.
-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-1: No redirect - Make no further changes to the webserver configuration.
-2: Redirect - Make all requests redirect to secure HTTPS access. Choose this for
-new sites, or if you're confident your site works on HTTPS. You can undo this
-change by editing your web server's configuration.
-```
-
-We recommend that you choose option 2, to redirect HTTP to HTTPS and always use a secure connection. You should be able to request your domain name with SSL as in `https://example.com` or `https://example.com/indexes`.
-
-#### 5.2.2. Option B: Custom SSL/TLS certificates
-
-When a `Certificate Authority` issues a SSL certificate for you, you receive at least two files with encrypted keys:
-
-+ Your **certificate** (commonly named `your_domain_name.pem` or `example.pem`)
-+ Your **key** (commonly named `your_domain_name.key` or `example.key`)
-
-`example.pem` and `example.key` will be used in the following examples. Make sure to replace `example` by the names of your own certificate files.
-
-All you need to do is store the certificate files in a secure location and use appropriate file system security permissions. Then, set the location of the certificates in Nginx configuration. It is also strongly recommended to redirect all HTTP requests to HTTPS (port 80 to 443).
-
-First, let's copy your certificate files in their conventional directory so the server can find them:
-
-```bash
-# Create a directory /etc/ssl/example to store the certificate files
-mkdir -p /etc/ssl/example
-
-# Move your files to /etc/ssl/example. We will suppose that your
-# files are called example.pem and example.key
-
-mv path-to-your-files/example.pem /etc/ssl/example/
-mv path-to-your-files/example.key /etc/ssl/example/
-```
+Enter your email address, agree to the Terms and Conditions, and choose your domain. When prompted if you want to automatically redirect HTTP traffic, choose option `2: Redirect`.
```

Comment:
We can add this external link to advertise them about the SSL renewal https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-debian-11#step-5-verifying-certbot-auto-renewal

Reply:
Great point. I forgot about the 90-day validity. I'll add an extra final step.